plugins/UM3NetworkPrinting/UM3InfoComponents.qml

```diff
@@ -119,11 +119,17 @@ Item
             onClicked: manager.loadConfigurationFromPrinter()
 
             function isClusterPrinter() {
```

Comment:
I'm not sure what the isClusterPrinter property is used for in the UI, but even a single printer running the cluster firmware should be treated as being in a cluster (since the API endpoints are different). If it's only UI fluffiness connected to this it's fine I guess.

Reply:
I'm not changing the logic here, just fixing the case where there is no connected printer at all.